tests/e2e/pageObjects/CreateSpacePage.js

```diff
@@ -0,0 +1,53 @@
+require("dotenv").config();
+
+const { format } = require("util");
+
+const config = require("../../cucumber.conf.js");
+
+class CreateSpacePage {
+
+  constructor() {
+    this.baseUrl = config.tandoorURL;
+    this.searchURL = this.baseUrl + `search/`;
+    this.spaceOverviewURL = this.baseUrl + `space-overview`;
+    this.createSpaceBtnSelector = `//input[@value="Create Space"]`;
+    this.spaceNameInputSelector = `#id_create-name`;
+    this.succesMsgSelector = `//div[@class="alert alert-success alert-dismissible fade show"]`;
+    this.errorMsgSelector = `//p[@id="error_1_id_create-name"]/strong`;
+    this.optionMenuSelector = `#navbarDropdownMenuLink`;
+    this.spaceNameSelector = `//a[text()="%s"]`;
+  }
+
+  async createSpace(spaceName) {
+    await page.locator(this.spaceNameInputSelector).fill(spaceName);
+    await page.locator(this.createSpaceBtnSelector).click();
+  }
+
+  async goToOverviewPage() {
+    await page.goto(this.spaceOverviewURL);
+  }
+
+  async submitRegisterData(username, password) {
+    await page.locator(this.usernameSelector).ill(username);
+    await page.locator(this.passwordSelector).fill(password);
+    await page.locator(this.cPasswordSelector).fill(password);
+    await page.locator(this.registerBtnSelector).click();
+  }
+
+  async getErrorMessage() {
+    return await page.innerText(this.errorMsgSelector);
+  }
```

Comment:
Do not wait but simply return the promise
```suggestion
  getErrorMessage() {
    return page.innerText(this.errorMsgSelector);
  }
```